app/Repositories/UserRepository.php

```diff
@@ -112,6 +113,23 @@ public function counts()
     }
 
     /**
+     * Get blog authors, their number of blogs, the last created date, and title of their latest blog entry.
+     *
+     * @return Illuminate\Database\Eloquent\Builder
+     */
+    public function getAuthorPostsWithOrder()
```

Comment:
This method name suggests to me it returns one author's posts in a certain order